src/fields/fft_poisson_solver/fft/WrapRocDST.cpp

```diff
@@ -0,0 +1,113 @@
+#include "AnyDST.H"
+#include "RocFFTUtils.H"
+#include "utils/HipaceProfilerWrapper.H"
+
+namespace AnyDST
+{
+    /** \brief Extend src into a symmetrized larger array dst
+     *
+     * \param[in,out] dst destination array, odd symmetry around 0 and the middle points in x and y
+     * \param[in] src source array
+     */
+    void ExpandR2R (amrex::FArrayBox& dst, amrex::FArrayBox& src)
+    {
+        HIPACE_PROFILE("AnyDST::ExpandR2R()");
+        /* todo */
```

Comment:
@MaxThevenet I left the bodies in this file unimplemented for now.
Please don't hesitate to push on this PR, it's probably a very similar implementation as in `WrapCuDST.cpp`.